tasks/56_search-and-replace/search-and-replace.js

```diff
@@ -0,0 +1,12 @@
+export function myReplace(str, before, after) {
```

Comment:
Please take a look at this https://spin.atomicobject.com/2011/04/10/javascript-don-t-reassign-your-function-arguments/